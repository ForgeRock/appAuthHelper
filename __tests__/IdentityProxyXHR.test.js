const IdentityProxyXHR = require("../IdentityProxyXHR");

describe("IdentityProxyXHR", () => {
    it("has the expected shape", () => {
        const resourceServers = ["https://test.com"];
        const transmissionPort = {
            postMessage: _ => _
        };
		
        expect(new IdentityProxyXHR(resourceServers, transmissionPort)).toMatchSnapshot();
    });	
});