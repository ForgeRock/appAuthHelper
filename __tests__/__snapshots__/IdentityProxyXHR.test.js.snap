// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdentityProxyXHR has the expected shape 1`] = `
Object {
  "failedRequestQueue": Object {},
  "resourceServers": Array [
    "https://test.com",
  ],
  "transmissionPort": Object {
    "postMessage": [Function],
  },
}
`;
